git-semver.sh

```diff
@@ -172,17 +172,22 @@ version-parse-pre-release() {
 version-get() {
     local sort_args version version_pre_releases pre_release_id_count pre_release_id_index
     local tags=$(git tag)
+    local sorted_tags=$(
+        echo "$tags" |
+            grep -oP "^${VERSION_PREFIX}\K[0-9]+\.[0-9]+\.[0-9]+.*" |
+            awk -F '[-+]' '{ print $1 }' |
+            uniq |
+            sort -t '.' -k 1,1n -k 2,2n -k 3,3n |
+            awk -v VERSION_PREFIX="${VERSION_PREFIX}" '{print VERSION_PREFIX $1}'
+    )
+
     local version_pre_release=$(
         local version_main=$(
-            echo "$tags" |
-                grep "^${VERSION_PREFIX}[0-9]\+\.[0-9]\+\.[0-9]\+" |
-                awk -F '[-+]' '{ print $1 }' |
-                uniq |
-                sort -t '.' -k 1,1n -k 2,2n -k 3,3n |
```

Comment:
For me it was enough to update the sort command and drop the `n`s from it, no other changes.